jigna/js/build.js

```diff
@@ -68,3 +69,37 @@ concatenate({
         'jigna'
     ]
 });
+
+// jigna-vue.js
+concatenate({
```

Comment:
I think it is much cleaner to have two separate files, there seems little point in keeping both.  `jigna.js` also starts up the angular app.  I don't know why we need that.  Calling `concatenate` twice simply creates each file and is perfectly legal.